web/static/js/study-responses.js

```diff
@@ -169,6 +169,11 @@ setupDataTableDates("individualResponsesTable", 3, "dateRangeFilter");
 // Toggle the filled/unfilled star image visibility on input checkbox state change
 $('.star-checkbox').change(function () {
     $(this).labels().children('.icon-star').toggleClass('icon-hidden');
+    $(this).each((_idx, el) => {
+        const td = el.parentElement;
+        td.dataset.sort = "False" == td.dataset.sort ? "True" : "False"
+        resp_table.rows().invalidate("dom").draw();
+    })
```

Comment:
@okaycj Do we also need to move this into the [change event callback](https://github.com/lookit/lookit-api/blob/3acdb72bfeb57ebd41c2e77e72ea7a6c73a922d2/web/static/js/study-responses.js#L161-L205), so that it's only triggered after a successful database update?
If so then I'll wait to review this (or let me know if I should make this change).

Reply:
Thank you!  This has been updated in commit [1909ee6](https://github.com/lookit/lookit-api/pull/1529/commits/1909ee620b51c97148d5c6e8fee47cc950d03cec).